app/Exceptions/Image/ImageStoreException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions\Image;
+
+use Exception;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class ImageStoreException extends Exception
+{
+    public function render(Request $request): JsonResponse
+    {
+        return new JsonResponse(
+            [
+                'error' => 'There was an error storing the image.',
+            ],
+            Response::HTTP_BAD_REQUEST,
+        );
```

Comment:
I know this is going into an API by default but should we wrap it in a `$request->wantsJson()` check just to safe?

Reply:
im going to say no to this as this is only ever going ot be an API that will be using JSON. if in the future we want to change it then we can look to make the change there. I feel like this is an early look ahead thats not needed at the moment.